util/dvsim/SimCfg.py

```diff
@@ -20,6 +21,63 @@
 from utils import VERBOSE, find_and_substitute_wildcards
 
 
+def pick_dump_format(fmts):
```

Comment:
My preference is to keep `SimCfg` as tool-agnostic as possible while remaining true to its primary job of supporting the simulation flow. Support for 2 additional simulators - Metrics DSim and Aldec Riviera were added purely through addition of HJson files. I would like that to continue being the case, rather than a user having to poke around the Python code to insert more tool-specific updates. 

I am ok with this change going in for now, but there are other ways of resolving / adding these checks elsewhere.

Reply:
Understood. I still think that if you're doing any non-trivial logic then it's clearer to do it in Python than hack it into the hjson files. I'll change this code to add a VPD default to the end of the list for now. That way, wave dumping will have a sane default for new tools that can use the lowest-common-denominator VPD format. Sadly, this means that `-t ascentlint -w` will silently work again.

Reply:
`LintCfg` does not use `args.waves` so that's harmless.

Reply:
Well, I guess this is a subjective thing. I probably prefer tools to shout at me if I say something silly ("Huh? Waves when linting?").